src/sidebar/store/modules/annotations.js

```diff
@@ -117,7 +125,8 @@ const initialState = {
 /** @typedef {typeof initialState} State */
 
 const reducers = {
-  ADD_ANNOTATIONS: function (state, action) {
+  /** @param {State} state */
+  ADD_ANNOTATIONS(state, action) {
```

Comment:
I haven't gone through and typed all the `action` parameters yet. That will happen in a separate pass.